cves/kernel/CVE-2021-30002.yml

```diff
@@ -84,14 +90,8 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: fb18802a338b36f675a388fc03d2aa504a0d0899
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, replace this comment with 'Manually confirmed'
+  note: Manually confirmed
 vcc_instructions: |
```

Comment:
You should check the commits and change the notes to manually confirmed just to make sure these are correct.

Reply:
Manually confirmed VCCs